uwac/libuwac/uwac-window.c

```diff
@@ -155,10 +157,11 @@ static void xdg_handle_configure(void* data, struct xdg_surface* surface,
 	}
 
 ack:
-	xdg_surface_ack_configure(surface, serial);
+	return;
 }
```

Comment:
lonely return ? (unneeded here)

Reply:
@hardening necessary for the label to jump to, did not want to refactor the whole function (but then again it is not that large)

Reply:
ok, done :)